src/test/java/Homework17.java

```diff
@@ -0,0 +1,38 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class Homework17 extends BaseTest {
+    public void choosePlaylist() {
+
+    }
+
+    public void clickAddBtn() throws InterruptedException {
+        WebElement addToBtn = driver.findElement(By.cssSelector("button.btn-add-to"));
+        addToBtn.click();
+        Thread.sleep(2000);
+    }
+
+    public void selectFirstSongResult() throws InterruptedException {
+        WebElement firstSongResult = driver.findElement(By.cssSelector("section#songResultsWrapper tr.song-item td.title"));
+        firstSongResult.click();
+        Thread.sleep(2000);
+
+    }
+
+    public void clickViewAllBtn() throws InterruptedException {
+        WebElement viewAllSearchResult = driver.findElement(By.cssSelector("div.results section.song h1 button"));
+        viewAllSearchResult.click();
+        Thread.sleep(2000);
+    }
+}
+
```

Comment:
Please, add Homework17 in TestNG.xml to run tests

For example

![image](https://user-images.githubusercontent.com/104761482/226163536-29b48154-2c7a-49f5-8ae7-fdd1f2e3c553.png)

Reply:
Thank you so much for taking the time to give me all this useful information. I really do appreciate it.